chart/kubeapps/templates/tiller-proxy-rbac.yaml

```diff
@@ -0,0 +1,36 @@
+{{- if .Values.rbac.create -}}
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: Role
+metadata:
+  name: {{ template "kubeapps.tiller-proxy.fullname" . }}
+  labels:
+    app: {{ template "kubeapps.tiller-proxy.fullname" . }}
+    chart: {{ template "kubeapps.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+rules:
+- apiGroups:
+  - ""
+  resources:
+  - secrets
+  verbs:
+  - get
```

Comment:
we can be careful here and only access to the secret `{{ template "kubeapps.tiller-proxy.fullname" . }}`

Reply:
Is this actually the case? 

I thought that tiller-proxy needs to access to any secret associated to every one of the repositories that have auth info no? https://github.com/kubeapps/kubeapps/blob/master/dashboard/src/actions/repos.ts#L123

Reply:
Yes, it needs to access all secrets because it will be reading the secret named after a added apprepository.

Reply:
oh, true, I forgot about the repository secrets. I thought this was just for the TLS secret

Reply:
The TLS secret is mounted in as a volume, so an RBAC role is not required for it.